libcnb-test/src/lib.rs

```diff
@@ -85,15 +86,37 @@ impl IntegrationTest {
     /// - When the connection to Docker failed
     /// - When the internal Tokio runtime could not be created
     pub fn new(builder_name: impl Into<String>, app_dir: impl AsRef<Path>) -> Self {
+        let tokio_runtime =
+            tokio::runtime::Runtime::new().expect("Could not create internal Tokio runtime");
+
+        let docker = match env::var("DOCKER_HOST") {
+            #[cfg(target_family = "unix")]
+            Ok(docker_host) if docker_host.starts_with("unix://") => {
+                Docker::connect_with_unix_defaults()
+            }
+            Ok(docker_host)
+                if docker_host.starts_with("tcp://") || docker_host.starts_with("https://") =>
+            {
+                Docker::connect_with_ssl_defaults()
+            }
+            Ok(docker_host) => panic!(
+                "Cannot connect to unsupported DOCKER_HOST '{}'",
```

Comment:
```suggestion
                "Cannot connect to unsupported DOCKER_HOST protocol '{}'",
```

Reply:
Thing is, that string would be more than just the protocol. Since this is so minor, I will skip this. :)